api/src/rustic_ai/api_server/guilds/router.py

```diff
@@ -76,6 +76,15 @@ def get_guild(guild_id: str, engine=Depends(Metastore.get_engine)) -> GuildSpecR
     return maybeGuild
 
 
+@router.post("/guilds/{guild_id}/relaunch", status_code=status.HTTP_204_NO_CONTENT, operation_id="relaunchGuild")
+def relaunch_guild(guild_id: str, engine=Depends(Metastore.get_engine)):
+    """
+    Relaunches a guild if is not already running.
+    """
+    guild_service.relaunch_guild(guild_id, engine)
+    return Response(status_code=status.HTTP_204_NO_CONTENT)
```

Comment:
we should return response with correct info i.e. if the guild was relaunched (200) or already running correctly (204), can help in frontend/clients, diplaaying right message aand waiting to launch, etc.